Dockerfile

```diff
@@ -0,0 +1,32 @@
+FROM phusion/baseimage:latest
+MAINTAINER Boik boik@tdohacker.org
+
+ENV HOME /root
+ARG DOMAIN
+ARG MODE=dev
+CMD ["/sbin/my_init"]
+
+WORKDIR $HOME
+RUN apt-get update \
+  && apt-get install -y rsync
+ADD . $HOME
+RUN chown root:root $HOME
+
+RUN ./extra/provision.sh -m $MODE -s `pwd`
+RUN [ "$MODE" = "prod" ] && rm /etc/nginx/sites-enabled/fbctf.conf || true
+
+COPY extra/http_fbctf.conf /etc/nginx/sites-available/http_fbctf.conf
+COPY extra/https_fbctf.conf /etc/nginx/sites-available/https_fbctf.conf
+RUN sed -i 's/domain.com/'$DOMAIN'/g' /etc/nginx/sites-available/http_fbctf.conf \
+  && sed -i 's/domain.com/'$DOMAIN'/g' /etc/nginx/sites-available/https_fbctf.conf
+
+RUN mkdir -p /etc/my_init.d
+COPY extra/setup_lets_encrypt.sh /etc/my_init.d/01_setup_lets_encrypt.sh
```

Comment:
You can just copy if it's in prod mode, you don't have to copy and delete later.